setup.md

```diff
@@ -174,44 +174,206 @@ before pushing them to GitHub.
 
 In order to preview changes locally, you must install the software described below.
 
-1.  **[Ruby](https://www.ruby-lang.org/en/downloads/)**.
-
-    **Linux/macOS**: Ruby is usually included with Linux and macOS ---
-    this is usually sufficient to preview your changes locally.
-
-    If you aren't able to preview reliably with your default Ruby installation,
-    you can use the same version of Ruby as GitHub. Currently,
-    GitHub uses Ruby 2.5.3. In order to install Ruby 2.5.3 on Linux and macOS, we recommend using
-    [rbenv](https://github.com/rbenv/rbenv):
-
+### Windows
+
+#### Option 1 - Using the Windows Subsystem for Linux (WSL)
+
+If your version of Windows supports it, using the WSL will make the installation of the tools needed easier. Instructions to install Linux distributions from Windows 10/Windows Server are available from the [Microsoft website](https://docs.microsoft.com/en-us/windows/wsl/about).
+
+Once you have installed a Linux distribution, you can follow the installation instructions for [Linux](#linux-ubuntu) listed below. If you install a distribution other than Ubuntu, you'll need to adjust the commands that install the packages. 
+
+#### Option 2 - Using Windows
```

Comment:
`Using Windows` is a rather general statement. Perhaps, "Using Windows built-in applications"...